doc/nrf/releases_and_maturity/migration/2.4.99-cs3_to_2.6.99-cs2/migration_guide_2.4.99-cs3_to_2.6.99-cs2_application.rst

```diff
@@ -0,0 +1,142 @@
+.. _migration_cs3_to_2_6_99_cs2_app:
+
+Migrate your application to |NCS| v2.6.99_cs2 (for v2.4.99-cs3 users)
+#####################################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This document describes the changes you should be aware of when migrating your application from |NCS| for the nRF54 customer sampling release v2.4.99-cs3 to |NCS| v2.6.99-cs2.
+
+.. _migration_cs3_to_2_6_99_cs2_app_overview:
+
+Overview
+********
+
+|NCS| v2.6.99-cs2 introduced a series of changes that might affect your existing applications.
+The following is a summary of the most important ones:
+
+Updated |NCS| toolchain
+  The |NCS| toolchain has been updated.
+  A bootstrap script specific to the nRF54H20 DK is now available to install and update the toolchain.
+  See :ref:`migration_cs3_to_2_6_99_cs2_env` for more info on how to upgrade your current |NCS| installation based on version 2.4.99-cs3.
+
+SDFW and SCFW
+  The Secure Domain Firmware (SDFW) and System Controller Firmware (SCFW) are no longer built from the source during the application build process, but they are installed as binaries from the provided firmware bundle and programmed when flashing the application to the device.
+  For additional details, see :ref:`migration_cs3_to_2_6_99_cs2_env_fwbundle`.
+
+nRF Util has now replaced nRF Command Line Tools.
```

Comment:
Perhaps a link to nRF Util download should be provided here? (https://www.nordicsemi.com/Products/Development-tools/nRF-Util)

Reply:
We have to wait with that. The current external production environment on artifactory is  different from this and needs to be adressed later.